beast/tools/helpers.py

```diff
@@ -355,12 +357,11 @@ def val_in_unit(varname, value, defaultunit):
     -------
     >>> r = 0.5
     >>> print(val_in_unit('r', r, 'degree'))
-    # UserWarning: Variable r does not have explicit units. Assuming `degree`
-    <Quantity(0.5, 'degree')>
+    0.5 degree
 
     >>> r = 0.5 * unit['degree']
     >>> print(val_in_unit('r', r, 'degree'))
-    <Quantity(0.5, 'degree')>
+    0.5 degree
     """
```

Comment:
We can add python 3 tests once the beast code runs in python 3.  Can't do that yet.